serveur.js

```diff
@@ -31,6 +44,7 @@ try {
 
   const PORT = argv.port ? argv.port : 8081;
   const SERVER = argv.server ? argv.server : os.hostname();
```

Comment:
ça ne fonctionne pas si on utilise docker car conflit entre id conteneur et localhost et dans swagger, pour toutes les requêtes j'ai  : 
Server response
Code | Details
-- | --
Undocumented | TypeError: Failed to fetch

Reply:
ça fonctionne si on ajoute "-h hostname" quand on lance le docker